pkgs/tools/security/verifpal/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, fetchFromGitHub, buildGoPackage, pigeon }:
+buildGoPackage rec {
+  pname = "verifpal";
+  version = "0.2";
+
+  goPackagePath = "github.com/SymbolicSoft/verifpal";
+  goDeps = ./deps.nix;
+
+  src = fetchFromGitHub {
+    owner = "SymbolicSoft";
+    repo = pname;
+    rev = version;
+    sha256 = "08a0xvgg94k6vq91ylvgi97kpkjbw0rw172v2dzwl2rfpzkigk1r";
+  };
+
+  postPatch = ''
+    sed -e 's|/bin/echo|echo|g' -i Makefile
```

Comment:
would it be neater to use a patch here? perhaps even patchShebangs would work

Reply:
the Makefile has a lot of `@/bin/echo "Some message"` lines. This will be more resilient to change when upstream adds more directives. I will make it even more resilient like this:

```diff
-    sed -e 's|/bin/echo|echo|g' -i Makefile
+    sed -e 's|/bin/echo |echo |g' -i Makefile
```

patchShebangs only works on the first line of the scripts